tests/Nelmio/Alice/support/fixtures/reference_range_1.yml

```diff
@@ -0,0 +1,8 @@
+\Nelmio\Alice\support\models\User:
+    user{1..3}:
+        uuid: <current()>
+        username: <username()>
+        fullname: <firstName()> <lastName()>
+        birthDate: '80%? <date> : 0000-00-00'
+        email: <email()>
+        favoriteNumber: 40%? <numberBetween(1, 99)>
```

Comment:
You would get an error if you haven't defined some fixtures matched by user* if a previous file.